src/File/ExcludeMimeType.php

```diff
@@ -22,6 +22,15 @@ class ExcludeMimeType extends MimeType
     const NOT_READABLE = 'fileExcludeMimeTypeNotReadable';
 
     /**
+     * @var array Error message templates
+     */
+    protected $messageTemplates = [
+        self::FALSE_TYPE   => "File has an incorrect mimetype of '%type%'",
```

Comment:
Constant was defined, the same constants name are in parent class `MimeType`, I've added only messages.
We can change it, maybe it is even better, but for consistency I'd prefer to keep `FALSE_TYPE` ...
And it's not in the scope of the issue ;)